docs/MPURegionsUsageGuide.md

```diff
@@ -132,18 +152,39 @@
 			void up_mpu_set_register(uint32_t *mpu_regs);
 			```
 3. **Example : Adding new MPU region, MPU_REG_NUM_MY_REGION, with RO setting**
-	1. Add your MPU region in enum MPU_REG_NUM in common mpu.h file.
+	1. Add the MPU region type in enum mpu_region_usages_e in common mpu.h file.
 	```c
-	enum MPU_REG_NUM {
+	enum mpu_region_usages_e {
 	 ....
-	 MPU_REG_NUM_MY_REGION,
-	 MPU_REG_NUM_MAX
+	 MPU_REGION_STACKOVF,
+	 MPU_REGION_MYREGION,
```

Comment:
Can you pls check it there is any way to highlight this line to show it is new line added?
Maybe we can make it bold.